test/libsolidity/SolidityParser.cpp

```diff
@@ -167,6 +167,90 @@ BOOST_AUTO_TEST_CASE(single_function_param)
 	BOOST_CHECK(successParse(text));
 }
 
+BOOST_AUTO_TEST_CASE(single_function_param_trailing_comma)
+{
+	char const* text = R"(
+		contract test {
+			function(uint a,) {}
+		}
+	)";
+	CHECK_PARSE_ERROR(text, "Unexpected trailing comma in arguments");
```

Comment:
I know this was the message suggested in the issue, but perhaps `Unexpected trailing comma in parameter list` is more universal given it is used on multiple places.

@chriseth @federicobond what do you think?

Reply:
Sure, let's go ahead with parameter list.